Cargo.toml

```diff
@@ -10,10 +10,9 @@ repository = "https://github.com/jakewilkins/gh-device-flow"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-reqwest = { version = "0.11", features = ["blocking", "json"] }
+reqwest = { version = "0.11", features = ["blocking", "json", "rustls-tls"] }
```

Comment:
Can you provide some more context here on why this change is required and what sort of validation you've done that it is non-breaking?

Reply:
Openssl-sys was taking a long time to compile on my end, so I found it could be replaced with rust-ls which seems to run better. Reqwest hasn't thrown any errors in the time that I've switched so I don't think there is much difference apart from performance.

Reply:
`rustls-tls` fails to compile correctly on the `arm` or `armv7` builds.

If you know of a simple way to make those work I'm glad to hear it, or if you're okay with merging without the `rustls-tls` change I'm happy to have that as well.

Reply:
If it still runs as normal without it that is good with me.

Reply:
I know that there is a way for me to make this work that would allow you to choose the one that's best for you but I haven't ever done that before and don't have the time to mess with figuring it right now 😅

Reply:
What do you mean?

Reply:
What I had in mind was something similar to https://github.com/jakewilkins/gh-device-flow/pull/3 but I just kind of feel like I'm misusing Cargo in some way and if I could find the right options things would go better.